mobile/lib/repositories/album_media.repository.dart

```diff
@@ -15,7 +16,9 @@ class AlbumMediaRepository implements IAlbumMediaRepository {
         await PhotoManager.getAssetPathList(
       hasAll: true,
     );
-    return assetPathEntities.map(_toAlbum).toList();
+    return assetPathEntities.map(_toAlbum).sortedBy((a) {
```

Comment:
This should be sorted from the page that needs the sorted info

Reply:
I couldn't think of any examples where you wouldn't want the album's in a "sane/expected order", hence putting it here. There's no real performance impact (since it's a cheap operation).
Let me know if you disagree and I'll move it.